rpc_util.go

```diff
@@ -231,6 +239,48 @@ func PerRPCCredentials(creds credentials.PerRPCCredentials) CallOption {
 	})
 }
 
+// CallContentSubtype returns a CallOption that will set the content-subtype
+// for a call. For example, if content-subtype is "json", the Content-Type
+// over the wire will be "application/grpc+json". See Content-Type on
+// https://grpc.io/docs/guides/wire.html#requests for more details. Note that
+// content-subtypes are case-insensitive.
+//
+// If CallCustomCodec is not also used, the content-subtype will be used to
+// look up the Codec to use in the registry controlled by RegisterCodec. See
+// the documention on RegisterCodec for details on registration. If no such
+// Codec is found, the call will result in an error with code codes.Internal.
```

Comment:
I'd mention here that the content subtype is a case-insensitive lookup, instead of above.

And above, mention that the subtype is converted to lower-case before being included in the content-subtype field.

Reply:
Done